File: app/src/main/java/com/checkpoint/qr/detector/events/OnPostProccessingCompletedEvent.java
package com.checkpoint.qr.detector.events;

import android.graphics.Bitmap;
import android.util.Pair;
import com.google.mlkit.vision.barcode.common.Barcode;
import com.checkpoint.qr.detector.pojos.DetectionResult;

import java.util.List;

public class OnPostProccessingCompletedEvent extends BaseEvent {

    public Pair<Bitmap, List<Barcode>> detectionResult;

    public OnPostProccessingCompletedEvent(Pair<Bitmap, List<Barcode>> detectionResult) {
        this.detectionResult = detectionResult;
    }

    public Pair<Bitmap, List<Barcode>> getResult() {

        return detectionResult;

    }
}

File: app/src/main/java/com/checkpoint/qrdetector/events/OnPostProccessingCompletedEvent.kt
package com.checkpoint.qrdetector.events

import android.graphics.Bitmap
import com.google.mlkit.vision.barcode.common.Barcode


class OnPostProccessingCompletedEvent(private  var detectionResult: Pair<Bitmap, MutableList<Barcode>>) : EventBase() {

    fun getResult(): Pair<Bitmap?, List<Barcode>?>? {
        return detectionResult
    }
}